lemmy_apub/src/inbox/community_inbox.rs

```diff
@@ -48,15 +49,15 @@ pub async fn community_inbox(
   })
   .await??;
 
-  if !community.local {
-    return Err(
-      anyhow!(
-        "Received activity is addressed to remote community {}",
-        &community.actor_id
-      )
-      .into(),
-    );
```

Comment:
This is already taken care of because `Community::read_from_name` now only reads local ones.